src/icons/glyphs/caffeineGlyph.ts

```diff
@@ -0,0 +1,4 @@
+import { CaffeineGlyph } from './svgs';
+import { IconProps, useIcon } from '../../shared-components/icon';
```

Comment:
```suggestion
import { type IconProps, useIcon } from '../../shared-components/icon';
```
Same suggestion for the other glyph components.